tests/unit-tests/test_ragstack.py

```diff
@@ -1,6 +1,6 @@
 def test_import():
-    from llama_index.vector_stores import AstraDBVectorStore  # noqa
-    from llama_index.vector_stores import CassandraVectorStore  # noqa
+    from llama_index.vector_stores.astra_db import AstraDBVectorStore  # noqa
```

Comment:
They removed it from the `__init__` ? That's a breaking change...

Reply:
YES ! 
that's why I want to cut ragstack 1.0.0...